sympy/ntheory/generate.py

```diff
@@ -142,7 +142,7 @@ def extend_to_no(self, i):
         while len(self._list) < i:
             self.extend(int(self._list[-1] * 1.5))
 
-    def primerange(self, a, b):
+    def primerange(self, a, b = None):
         """Generate all prime numbers in the range [a, b).
```

Comment:
Update the docs to notify that `b` is optional now.

Reply:
It should probably be explained by analogy with `range`